### PR TITLE
feat(frontend): first draft implementation for new all campaigns page

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/frontend/src/components/campaigns/pool/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { Typography } from "@/src/ui/typography";
+import styles from "./styles.module.css";
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/pool/index.tsx:4]: I'd keep the styles as the last import like the other components.

File: packages/frontend/src/components/campaigns/rewards/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"use client";
+
+import { Typography } from "@/src/ui/typography";
+import styles from "./styles.module.css";
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/rewards/index.tsx:4]: Same here, and for the others below.

File: packages/frontend/src/components/campaigns/pool/index.tsx (hunk 1 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/pool/index.tsx]: Addressed in 1cc81e200b9035b50c9a7c9f8a5721dbae9926bd

File: packages/frontend/src/components/campaigns/rewards/index.tsx (hunk 2 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/rewards/index.tsx]: Addressed in 1cc81e200b9035b50c9a7c9f8a5721dbae9926bd

**Review by guerrap — APPROVED** [review]

**Pull request merged**